pallets/runtime/common/src/runtime.rs

```diff
@@ -951,7 +957,17 @@ macro_rules! runtime_apis {
                     value: Balance
                 ) -> polymesh_primitives::asset::GranularCanTransferResult
                 {
-                    Asset::unsafe_can_transfer_granular(from_custodian, from_portfolio, to_custodian, to_portfolio, ticker, value)
+                    let mut weight_meter = WeightMeter::max_limit();
+                    Asset::unsafe_can_transfer_granular(
+                        from_custodian,
+                        from_portfolio,
+                        to_custodian,
+                        to_portfolio,
+                        ticker,
+                        value,
+                        &mut weight_meter
+                    )
+                    .unwrap()
```

Comment:
Reminder: we have to remove this

Reply:
I added an RPC endpoint that will return the consumed weight, but excluding the `update_statistics` function. To include that, we can either add a parameter to the function (in order not to change any data) or write a separate function that only calculates its weight. 

I added a new field for the consumed weight to the `GranularCanTransferResult` (won't that require bumping the version as well?) . I have also added a comment on [MESH-2008] to return the errors and bump the APIs version.